src/Core/src/Fonts/FontRegistrar.cs

```diff
@@ -117,71 +106,5 @@ bool IsFile(string path, string file)
 
 			return path.Replace(file, "").EndsWith(".", StringComparison.Ordinal);
 		}
-
-		bool CanUsePackagedNativeFonts =>
-#if WINDOWS
-			true;
-#else
-			false;
-#endif
```

Comment:
This is genuinely cause for concern for all the other devs out there. Which dummy wrote this instead of using the partial classes to actually do different logic. Me. 😭